src/css/profile/wearable/changeable/theme-circle/arclistview.less

```diff
@@ -64,7 +64,7 @@
 		display: flex;
 		align-items: center;
 
-		&:not(.ui-li-group-index):not(.ui-listview-divider) {
+		&:not(.ui-li-group-index):not(.ui-listview-divider):not(.li-has-multiline) {
 			min-height: 108 * @unit_base;
```

Comment:
unit_base or px_base? what is correct?

Reply:
We decided to gradually introduce px_base to wearable profile as well. So to answer your question, yes we should use px_base. However, I suggest applying them to lines where code is changed not to make refactoring commits.

Reply:
to avoid confusion I put this information on wiki
https://github.com/Samsung/TAU/wiki#using-x_base-in-css